views/edit-student.ejs

```diff
@@ -45,46 +43,53 @@
 				<label for="interestsAndHobies">Hobbies/Interests.</label>
 				<input type="text" class="form-control" name="interestsAndHobies" value="<%=student.interestsAndHobies%>" <% if (view) { %> readonly <% } %>>
 			</div>
+			<div class="form-group col-md-6">
+				<label for="grade">Grade</label>
+				<select name="grade" class="form-control" required <% if (view) { %> readonly <% } %>>
+					<% let grades = ["6th", "7th", "8th", "9th", "10th", "11th", "12th", "College Freshman", "College Sophmore", "College Junior", "College Senior"]; %>
+					<% for (let i = 0; i < grades.length; i++) { %>
+						<option <% if (student.grade == grades[i]) { %> selected <% } %>><%= grades[i] %></option>
+					<% } %>
+				</select>
+			</div>
 			<div class="form-group col-md-6">
 				<label for="school" class="form-label">School</label>
 				<% let schools = ["Berea-Midpark High School", "Solon High School", "Lakewood High School", "Westlake High School"]; %>
 				<input id="school"  class="form-control" name="school" list="school_options" required <% if (!add) { %> value="<%= student.school %>" <% } %> <% if (view) { %> readonly <% } %>/>
 				<datalist id="school_options">
 					<% for (let i = 0; i < schools.length; i++) { %>
-						<option value="<%= schools[i] %>">
+							<option value="<%= schools[i] %>">
 					<% } %>
 				</datalist>
 			</div>
-		</div>
-		<div class="form-row">
 			<div class="form-group col-md-6">
-				<label for="grade">Grade</label>
-				<select name="grade" class="form-control" required <% if (view) { %> readonly <% } %>>
-					<% let grades = ["6th", "7th", "8th", "9th", "10th", "11th", "12th", "College Freshman", "College Sophmore", "College Junior", "College Senior"]; %>
-					<% for (let i = 0; i < grades.length; i++) { %>
-						<option <% if (student.grade == grades[i]) { %> selected <% } %>><%= grades[i] %></option>
+				<label for="program_list">Programs Involved In</label>
+				<select class="form-control" name="program_list" id="program_list" multiple <% if (view) { %> readonly <% } %>>
+					<% for (let i = 0; i < programs.length; i++) { %>
+						<% if (view) { %>
+							<option><%= programs[i] %></option>
+						<% } else { %>
+							<option><%= programs[i].title %> - <%= programs[i].id %></option>
```

Comment:
When editing, is it possible to make the currently enrolled programs selected in the box?